syntax = "proto3";
package alert_event;





/*
GetAlertEventNotRecoverGroupByObject请求
*/
message GetAlertEventNotRecoverGroupByObjectRequest {
    /*
    告警模型id，多个用逗号分隔
    */
    repeated string objectIds = 1 ;
    /*
    开始时间, e.g.: -1h 过去1小时, -1d 过去1天, -30m 过去30分钟, 1490967693 unix 时间戳
    */
    string st = 2 ;
    /*
    结束时间, e.g.: -1h 过去1小时, -1d 过去1天, -30m 过去30分钟, 1490967693 unix 时间戳。默认为当前时间
    */
    string et = 3 ;
    /*
    sdk限制字段使用
    */
    repeated string XXX_RestFieldMask = 4 ;
}

/*
GetAlertEventNotRecoverGroupByObject返回
*/
message GetAlertEventNotRecoverGroupByObjectResponse {
message List {
    /*
    模型ID
    */
    string objectId = 1 ;
    /*
    告警数量
    */
    int32 count = 2 ;
}
        
    
    /*
    总数
    */
    int32 total = 1 ;
    /*
    统计列表
    */
    repeated List list = 2 ;
}

/*
GetAlertEventNotRecoverGroupByObjectApi返回
*/
message GetAlertEventNotRecoverGroupByObjectResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    alert_event.GetAlertEventNotRecoverGroupByObjectResponse data = 4 ;
}

